src/containers/pages/MDAPoint/SchoolReports/index.tsx

```diff
@@ -0,0 +1,209 @@
+import reducerRegistry from '@onaio/redux-reducer-registry';
```

Comment:
I think we shouldn't mention "school" in this module.  It really is not about "schools", it is about OpenSRP locations.  It just so happens that for MDA in Eswatini, the locations are schools.  So instead of calling this SchoolReportsList, I would call it LocationReportsList.